bin/bb/cmd/dump.go

```diff
@@ -50,63 +39,26 @@ func newDumpCmd() *cobra.Command {
 				defer f.Close()
 				out = f
 			}
-			return dumpDatabase(context.Background(), databaseType, username, password, hostname, port, database, out, tlsCfg, schemaOnly)
+			return dumpDatabase(context.Background(), u, out, schemaOnly)
 		},
 	}
-	dumpCmd.Flags().StringVar(&databaseType, "type", "mysql", "Database type. (mysql or pg).")
-	dumpCmd.Flags().StringVar(&username, "username", "", "Username to login database. (default mysql:root pg:postgres).")
-	dumpCmd.Flags().StringVar(&password, "password", "", "Password to login database.")
-	dumpCmd.Flags().StringVar(&hostname, "hostname", "", "Hostname of database.")
-	dumpCmd.Flags().StringVar(&port, "port", "", "Port of database. (default mysql:3306 pg:5432).")
-	dumpCmd.Flags().StringVar(&database, "database", "", "Database to connect and export.")
-	dumpCmd.Flags().StringVar(&file, "file", "", "File to store the dump. Output to stdout if unspecified")
-
-	// tls flags for SSL connection.
-	dumpCmd.Flags().StringVar(&sslCA, "ssl-ca", "", "CA file in PEM format.")
-	dumpCmd.Flags().StringVar(&sslCert, "ssl-cert", "", "X509 cert in PEM format.")
-	dumpCmd.Flags().StringVar(&sslKey, "ssl-key", "", "X509 key in PEM format.")
 
+	dumpCmd.Flags().StringVar(&dsn, "dsn", "", "database connection string. e.g. mysql://root@localhost:3306/bytebase")
```

Comment:
should include the password as the example

Reply:
user is likely not able to get it right on first shot, so when they use bb --help, the help should be clear.

e.g. the ssl flag should also be present here otherwise users will have a hard time to figure out.